PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -41,6 +42,9 @@ def makePuppiesFromMiniAOD( process ):
     process.puppiForMET = process.puppiPhoton.clone()
     setupPuppiPhotonMiniAOD(process)
     #Line below doesn't work because of an issue with references in MiniAOD without setting useRefs=>False and using delta R
-    #process.puppiForMET.puppiCandName    = 'puppiMerged'
-    #process.puppiForMET.useRefs          = False
+    process.puppiForMET.puppiCandName    = 'puppiMerged'
+    process.puppiForMET.useRefs          = False
 
+    #making a sequence for people running the MET tool in scheduled mode
+    puppiMETSequence = cms.Sequence(process.puppi*process.pfLeptonsPUPPET*process.pfNoLepPUPPI*process.puppiNoLep*process.puppiMerged*process.puppiForMET)
```

Comment:
scheduled mode use should be discouraged.
Please make this sequence creation optional (controlled by a parameter).